### PR TITLE
Increase debounce time to make sure controls are resized after the sidebar disappeared

### LINKED ISSUE #2346: Gallery button stays on position even if sidebar is not longer opened
### Expected behaviour
After the sidebar was closed, the gallery icon goes back to the the right side.

### Actual behaviour
Sadly it does not :(

### Server configuration

**Nextcloud version:** (see Nextcloud admin page)
11 beta

### Client configuration
**Browser:**
Safari 
**Operating system:**
macOS Sierra

### Screenshots

With Sidebar:
correct_1

Without Sidebar:
incorrect_1

### PR DESCRIPTION
**Pull request by MariusBluem**
Follow up of #2365 - Fixes #2346 😉 Still not perfect with Safari, but better than now 😅 It is now jumping back after some ms 😁 

Signed-off-by: Marius Blüm <marius@lineone.io>

### CODE REVIEW

File: core/js/js.js [hunk 1]
```diff
@@ -1502,7 +1502,7 @@ function initCore() {
 
 		$(window).resize(_.debounce(adjustControlsWidth, 250));
 
-		$('body').delegate('#app-content', 'apprendered appresized', _.debounce(adjustControlsWidth, 100));
+		$('body').delegate('#app-content', 'apprendered appresized', _.debounce(adjustControlsWidth, 500));
```
Comment (MariusBluem) [inline core/js/js.js]: In 50-steps or is 100 enough?

Comment (nickvergessen) [inline core/js/js.js]: 50 or 10? :P I don't care just make it fast

**Comment by LukasReschke** [comment]
@MariusBluem Can we move that to 12? I'd prefer to consider 11 closed for any non-critical merges.

File: core/js/js.js (hunk 1 above)
Comment (MorrisJobke) [inline core/js/js.js]: For me this works with 150 -.- ... looks like a weird race condition. I tried to debug this: first I tried that the `adjustControlsWidth` is called too early - but this is somehow not the case.

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
@MariusBluem @LukasReschke @georgehrke Can you give this a try in Safari? (having the gallery installed and open&close the sidebar in files app should result in the gallery switcher icon on the far right)

Thanks

**Comment by MorrisJobke** [comment]
@nickvergessen increasing the debounce time helped, but I have no idea why :(

**Comment by MorrisJobke** [comment]
> @MariusBluem @LukasReschke @georgehrke Can you give this a try in Safari? (having the gallery installed and open&close the sidebar in files app should result in the gallery switcher icon on the far right)

Ping ;)

**Review by LukasReschke — APPROVED** [review]
Should be okay :)

**Pull request merged**